.github/workflows/testing.yml

```
@@ -39,25 +39,27 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [ "3.7", "3.8" , "3.9", "3.10" ]
-        django-version: [ "3.0", "3.1", "3.2", "4.0" ]
+        django-version: [ "3.0", "3.1", "3.2", "4.0", "4.1" ]
```

NIP: I would leave here only the newest and LTS versions:

```suggestion
        django-version:
          - "3.2"
          - "4.0"
          - "4.1"
```

Reference: https://www.djangoproject.com/download/ - `Supported Version` section

---

NIP: it will be also nice to include 2 additional tests, that could fail:

+ `python-version: 3.11-dev` and `django-version: "4.1"`
+ `python-version: 3.10` and `django-version` installed from the `main` branch

---

I think the main branch is a little too much. Since Django puts out release candidates I think testing those should be enough, unless someone explicitly asks us to check main.

But agree we need to add 4.1, 3.11.0-rc.1 and can remove older v3s :slightly_smiling_face: